azure-auth-helper/src/main/java/com/microsoft/azure/auth/configuration/AuthMethod.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.auth.configuration;
+
+/**
+ * Enum records detail auth method
+ */
+public enum AuthMethod {
+    AZURE_CLI,
```

Comment:
where is azure auth plugin?

Reply:
Divided into 3 parts:
- AZURE_SECRET_FILE
- OAUTH
- DEVICE_LOGIN